src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/IndexRequest.cs

```diff
@@ -19,5 +19,6 @@ public int PageNumber
         public string SelectedCourse { get; set; }
         public DateTime? SelectedEndDate { get; set; }
         public ApprenticeshipStatus? SelectedStatus { get; set; }
+        public bool FromSearch { get; set; }
```

Comment:
shouldn't this be nullable?

Reply:
We only need to know that its true to kick off the cookie search so making it null isn't really needed as it'll default to false